### ISSUE TITLE
loaded your experimental builld 2276, now stick on that

### ISSUE THREAD

**Issue by moyesboy** [fhorinek/SkyDrop#163]
after loading 2276, no other firmware will load.
always error #181 on reboot.
2276 remains loaded and running.

Tried: several earlier firmwares. Same error #181.
removing battery plug, shorting +/- on pcb when power is removed. no luck
loading file direct to sd card using pc. - no luck
deleting all files form sd and putting only the skydrop.fw on it.  - no luck.

What to do?
Seems 2276 is working but for this strang 1970 date thing? would like to use it tomorrow.

**Comment by moyesboy** [comment]
went to spare unit (kathleens). this is running 2256.
Loading 2288 has same error #181.
This is odd. not sure what to do now. is anyone successfully loading 2288?
failure to load other softwares on my other unit is unexplained. dont want to "break" thins one too - though perhaps I already have...

**Comment by fhorinek** [comment]
error #181 during update means that the update file is corrupted.
Can you send me the update file you are using?

**Comment by moyesboy** [comment]
Ah, I seem to have solved this.
I am using chrome browser. And I made many successful updates, but this time I only got 49k file (repeatedly).
not sure what I am doing differently...I had this problem ages ago using IE but never had it since using chrome. It looks like a need a key by key instruction on how to access the file correctly!
I anaged to do it now fiddling about - cant really remember what I did though!
